plugin/src/App/StatusPages/Settings/Setting.lua

```diff
@@ -59,6 +59,7 @@ function Setting:render()
 			LayoutOrder = self.props.layoutOrder,
 			ZIndex = -self.props.layoutOrder,
 			BackgroundTransparency = 1,
+			Visible = if self.props.visible == nil then true else self.props.visible,
```

Comment:
```suggestion
			Visible = self.props.visible,
```

Reply:
For some reason when writing this I must have thought nil being falsy meant that no prop passed would default to not visible. I was clearly very tired, since nil means there's no prop and we resort to the Roblox default of visible. Thanks for noticing!